gdnative-derive/src/native_script.rs

```diff
@@ -49,6 +49,10 @@ pub(crate) fn impl_empty_nativeclass(derive_input: &DeriveInput) -> TokenStream2
 }
 
 pub(crate) fn derive_native_class(derive_input: &DeriveInput) -> Result<TokenStream, syn::Error> {
```

Comment:
This wrapper seems to unnecessary?
I found that other function in `gdnative_derive` (e.g. `methods::derive_methods`, `variant::derive_to_variant`, etc.) returns a `Result<proc_macro2::TokenStream, syn::Error>`, and convert the result into `proc_macro::TokenStream` in `lib.rs`.
I think it might be more consistent if `native_script::derive_native_class` also returns `proc_macro2::TokenStream`?